drush/database_sanitize.drush8.inc

```diff
@@ -30,15 +30,16 @@ function database_sanitize_drush_help($section) {
  * Implements hook_drush_command().
  */
 function database_sanitize_drush_command() {
+  $sanitize_filename = 'database.sanitize.yml';
```

Comment:
To be honest, I would be fine without mentioning here the actual database file name to avoid duplicating this, but its not the end of the world, so lets move on :+1: